lib/oga/xml/document.rb

```diff
@@ -57,6 +57,12 @@ def root_node
         self
       end
 
+      # Returns the top level element in the document.
+
```

Comment:
I'll stop pushing on `Element#<<`.  In that case, I agree that it's subjective, and as I stated at the beginning, I could argue either way on it.

However, I believe not having an accessor for the root element of the document is literally an API bug.  I don't mean simply all of the other 3 major Ruby XML implementations support that, I've literally never seen an XML API **in any language** that did not.

- Python: [xml.dom.Document.documentElement](https://docs.python.org/2/library/xml.dom.html#xml.dom.Document.documentElement)
- Java: [org.w3c.dom.Document.getDocumentElement](https://docs.oracle.com/javase/7/docs/api/org/w3c/dom/Document.html#getDocumentElement())
- PHP: [DOMDocument::$documentElement](http://php.net/manual/en/class.domdocument.php#domdocument.props.documentelement)
- Scala: [scala.xml.Document.docElem](http://www.scala-lang.org/api/2.10.4/index.html#scala.xml.Document@docElem:Node)

However, I'll roll that out to a separate pull request with some approximation of what I think the correct HTML and XML behavior would be.